Configuration/Production/OneginiConfigModel.m

```diff
@@ -2,30 +2,30 @@
+	return @[@"MIIEkDCCA3igAwIBAgIRAIK3ZQXQzIr6cowSqEK3qOowDQYJKoZIhvcNAQELBQAwTDEgMB4GA1UECxMXR2xvYmFsU2lnbiBSb290IENBIC0gUjMxEzARBgNVBAoTCkdsb2JhbFNpZ24xEzARBgNVBAMTCkdsb2JhbFNpZ24wHhcNMjUwMTIyMDQwNjI3WhcNMjcwMTIyMDAwMDAwWjBYMQswCQYDVQQGEwJCRTEZMBcGA1UEChMQR2xvYmFsU2lnbiBudi1zYTEuMCwGA1UEAxMlR2xvYmFsU2lnbiBBdGxhcyBSMyBEViBUTFMgQ0EgMjAyNSBRMjCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAMyoB0IupG7uesoJu04ahMOS+4z34aqYj7C2lMDbbPOWZSmfnhC8fnoWA8Fh/2A6HmkiZNiDEwbe31MvxqKtqPJeafhrpmDACZ6pRQ9I172rrxfTeFlmaUzh2kaVWPwl+rFuFcMoJzFpcKNZiaVR3SRsARka5S2BENu/C46oJ8cNTWwa6UfBLHWnexaAO9ek/vSIUYQHZN1TdJV85nq0nwOKhxTCaDRAtSWNKqp+hB7Bt70CpfeQZ5kNUsvTv8nM2Zrft2WXGTEjeKHAqDhl4K+SDBYtFVVcAMAP5agS2G5Pi6RJMm8aWnIWt0Jr45pvlyRRFy7/AoQDmt8YeHVDAc8CAwEAAaOCAV8wggFbMA4GA1UdDwEB/wQEAwIBhjAdBgNVHSUEFjAUBggrBgEFBQcDAQYIKwYBBQUHAwIwEgYDVR0TAQH/BAgwBgEB/wIBADAdBgNVHQ4EFgQUnPtVERBAoj9Fed1cLoJjk0AWR8owHwYDVR0jBBgwFoAUj/BLf6guRSSuTVD6Y5qL3uLdG7wwewYIKwYBBQUHAQEEbzBtMC4GCCsGAQUFBzABhiJodHRwOi8vb2NzcDIuZ2xvYmFsc2lnbi5jb20vcm9vdHIzMDsGCCsGAQUFBzAChi9odHRwOi8vc2VjdXJlLmdsb2JhbHNpZ24uY29tL2NhY2VydC9yb290LXIzLmNydDA2BgNVHR8ELzAtMCugKaAnhiVodHRwOi8vY3JsLmdsb2JhbHNpZ24uY29tL3Jvb3QtcjMuY3JsMCEGA1UdIAQaMBgwCAYGZ4EMAQIBMAwGCisGAQQBoDIKAQMwDQYJKoZIhvcNAQELBQADggEBAAH2bqL3uf9QrGd0o+R5qzRHEsLsXA3Ygc/vwcAjZ3Dzctw2dKfZftQN6PA82HjXzPKM1+FAcjdz5DXQKJqqIBFPFSdQkZknBQDguNQ+zRpI4RnXncmKw1eBJEgSDSxd9EnWqt2OULs0pt67DXzA7ku/kGBitVWEWLmiSjZXfpkVtwSZtNux6Vr92wy9wzBu6Ze/MGRxmhmy6DYlybDsL2pAlRANTtVOVcO0Znq4IaE8UbY8N70nXaqVAlNykgFZogIABeEiY4T3LVDUKC9LOLKiA6ic9ygfVwSb/3wQm3e8raR5jHaoobE+v1dn9rCxc7BFtBwVvbO81i9DsdCnYD4="]; //Base64Certificates
 }
 
 + (NSDictionary *)configuration
 {
     return @{
              @"ONGServerType" : @"access",
-             @"ONGServerVersion" : @"1.61.0",
+             @"ONGServerVersion" : @"1.66.2",
              @"ONGAppIdentifier" : @"ExampleApp",
              @"ONGAppPlatform" : @"ios",
-             @"ONGAppVersion" : @"6.3.4",
-             @"ONGAppBaseURL" : @"https://mobile-security-proxy.onegini.com",
-             @"ONGResourceBaseURL" : @"https://mobile-security-proxy.onegini.com/resources/",
+             @"ONGAppVersion" : @"6.3.5",
```

Comment:
Can we have 6.4.0 due to app identifier change? It is significant. We did it between 6.2.x -> 6.3.x

Reply:
no we cannot, everywhere is set to 635, api does not change so no worries